--- releases-2.1.1/setup.py.orig	2023-02-16 20:51:03.000000000 +0100
+++ releases-2.1.1/setup.py	2025-12-01 21:47:52.017112105 +0100
@@ -32,7 +32,7 @@ setup(
         # Continuing to pin an old semantic_version until I have time to update
         # and finish the branch I made for
         # https://github.com/bitprophet/releases/pull/86#issuecomment-580037996
-        "semantic_version<2.7",
+        "semantic_version<3",
     ],
     python_requires=">=3.6",
     classifiers=[
